doc/source/management/data_at_rest_encryption.rst

```diff
@@ -190,6 +190,32 @@ System Variables
 
 The variable turns on binary and relay logs encryption.
 
+Temporary file encryption
+=========================
+
+A new feature, implemented since |Percona Server| :rn:`5.7.22-22`, is
+encryption of temporary files, triggered by the :variable:`encrypt-tmp-files`
+option.
+
+For each temporary file, an encryption key is generated locally, only kept
+in memory for the lifetime of the temporary file, and discarded afterwards.
+The implementation ensures that local per-file key generation is cheap enough
```

Comment:
Please remove `The implementation ensures that local per-file key generation is cheap enough even in the case of high rate of temporary file creation.` - these are internal details.